examples/streamed/streamed-server.js

```diff
@@ -1,54 +1,68 @@
-var express = require('express');
+var express = require("express");
```

Comment:
if it's not much trouble can you commit your vscode settings as well so we can stay in sync?  I'm not sure I'm using anything, as evidenced by the inconsistent nature of this file.

Reply:
I added `.prettierrc.json` with my configuration